Enhancements/use profile image

JayyajGH opened the pull request:
The next round of using the profile image component

aaronferris commented on components/MyMessageReply.vue:
```diff
@@ -161,3 +134,22 @@ export default {
+.snippet {
+  border: 1px solid $color-gray--light;
+  border-radius: 10px;
+  padding-top: 2px;
```
Could short hand this

aaronferris commented on components/ProfileImage.vue:
```diff
@@ -77,4 +77,14 @@ export default {
+  width: 100px;
+  height: 100px;
+
+  @include media-breakpoint-up(md) {
```
Feels slightly odd to have a media query setting the same value as the default

JayyajGH replied:
The image was 100px in all breakpoints before but I've now changed it to be 50px on smaller screens and 100px on larger screens.  This whole section needs some work though as there are existing layout issues that I haven't touched for now.

aaronferris commented on pages/settings/index.vue:
```diff
@@ -824,3 +796,19 @@ export default {
+@import 'color-vars';
+
+.groupprofile {
+  height: 100px !important;
```
Can we avoid the !important?

aaronferris commented on pages/settings/index.vue:
```diff
@@ -824,3 +796,19 @@ export default {
+}
+
+.nocardbot .card-body {
+  padding-bottom: 0px;
```
Dont need to specify 'px' just '0' will do.

JayyajGH commented:
@aaronferris - At the moment I've just replaced the profile image code with my new component.  I haven't refactored any of the old existing code in the components I've changed.  Although I agree with your comments I'm sort of loathe to make lots of random changes and increase the scope as part of this PR.  I don't even know if half of those old classes are still used.

I'll look at the media query one for the XL breakpoint as I think that is incorrect.

edwh commented:
Lovely to see some crossreviewing happening, since you two are much 
better able to review each other's CSS than I am.

Agree not to fix too much creep stuff, but if appropriate please add to 
the shrapnel issue.

Where will the profile shrink in this fix?  There are some cases (like 
head of a chitchat thread) where it shouldn't be too small.


------ Original Message ------
From: "JayyajGH" <notifications@github.com>
To: "Freegle/iznik-nuxt" <iznik-nuxt@noreply.github.com>
Cc: "Subscribed" <subscribed@noreply.github.com>
Sent: 06/01/2020 10:21:42
Subject: Re: [Freegle/iznik-nuxt] Enhancements/use profile image (#157)

>@JayyajGH commented on this pull request.
>
>--------------------------------------------------------------------------------
>In components/ProfileImage.vue 
><https://github.com/Freegle/iznik-nuxt/pull/157#discussion_r363235006>:
>
> > @@ -77,4 +77,14 @@ export default {
>      height: 50px;
>    }
>  }
>+
>+.profile--xl {
>+  width: 100px;
>+  height: 100px;
>+
>+  @include media-breakpoint-up(md) {
>
>The image was 100px in all breakpoints before but I've now changed it 
>to be 50px on smaller screens and 100px on larger screens. This whole 
>section needs some work though as there are existing layout issues that 
>I haven't touched for now.
>
>—
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly, view it on GitHub 
><https://github.com/Freegle/iznik-nuxt/pull/157?email_source=notifications&email_token=AAAN26EYTAS4HUJKFIHPRHDQ4MA3NA5CNFSM4KDCBQDKYY3PNVWWK3TUL52HS4DFWFIHK3DMKJSXC5LFON2FEZLWNFSXPKTDN5WW2ZLOORPWSZGOCQXBZGY#discussion_r363235006>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AAAN26HJODVFKGIUDSNMIJDQ4MA3NANCNFSM4KDCBQDA>.
>

JayyajGH commented:
Yeah it's always good to have everything reviewed.  It makes you question your solution more thoroughly. 

The only size I changed from what it what originally is at the top of the settings page.  See what you think...

edwh commented:
I'll have a look.  Ready to merge?

------ Original Message ------
From: "JayyajGH" <notifications@github.com>
To: "Freegle/iznik-nuxt" <iznik-nuxt@noreply.github.com>
Cc: "Edward Hibbert" <edward@ehibbert.org.uk>; "Comment" 
<comment@noreply.github.com>
Sent: 06/01/2020 11:34:14
Subject: Re: [Freegle/iznik-nuxt] Enhancements/use profile image (#157)

>Yeah it's always good to have everything reviewed. It makes you 
>question your solution more thoroughly.
>
>The only size I changed from what it what originally is at the top of 
>the settings page. See what you think...
>
>—
>You are receiving this because you commented.
>Reply to this email directly, view it on GitHub 
><https://github.com/Freegle/iznik-nuxt/pull/157?email_source=notifications&email_token=AAAN26H6WT3GRR5OAXAV75DQ4MJLNA5CNFSM4KDCBQDKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFGNKQ#issuecomment-571106986>, 
>or unsubscribe 
><https://github.com/notifications/unsubscribe-auth/AAAN26D2NITPW64DVKZE7VTQ4MJLNANCNFSM4KDCBQDA>.
>

edwh commented:
I think it looks a bit small on mobile now, there.


------ Original Message ------
From: "Edward Hibbert" <edward@ehibbert.org.uk>
To: "Freegle/iznik-nuxt" 
<reply@reply.github.com>
Sent: 06/01/2020 11:40:05
Subject: Re[2]: [Freegle/iznik-nuxt] Enhancements/use profile image 
(#157)

>I'll have a look.  Ready to merge?
>
>------ Original Message ------
>From: "JayyajGH" <notifications@github.com>
>To: "Freegle/iznik-nuxt" <iznik-nuxt@noreply.github.com>
>Cc: "Edward Hibbert" <edward@ehibbert.org.uk>; "Comment" 
><comment@noreply.github.com>
>Sent: 06/01/2020 11:34:14
>Subject: Re: [Freegle/iznik-nuxt] Enhancements/use profile image (#157)
>
>>Yeah it's always good to have everything reviewed. It makes you 
>>question your solution more thoroughly.
>>
>>The only size I changed from what it what originally is at the top of 
>>the settings page. See what you think...
>>
>>—
>>You are receiving this because you commented.
>>Reply to this email directly, view it on GitHub 
>><https://github.com/Freegle/iznik-nuxt/pull/157?email_source=notifications&email_token=AAAN26H6WT3GRR5OAXAV75DQ4MJLNA5CNFSM4KDCBQDKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIFGNKQ#issuecomment-571106986>, 
>>or unsubscribe 
>><https://github.com/notifications/unsubscribe-auth/AAAN26D2NITPW64DVKZE7VTQ4MJLNANCNFSM4KDCBQDA>.
>>

JayyajGH commented:
I'll bump it up to 75px then... And then it'll be ready to merge.

JayyajGH commented:
@edwh - It's now slighter bigger on mobile

JayyajGH commented:
I've added all of Aaron's comments to the shrapnel issue
